src/rust/engine/async_semaphore/src/tests.rs

```diff
@@ -180,17 +180,17 @@ fn dropped_future_is_removed_from_queue() {
     .expect("thread1 didn't acquire.");
   let waiter = handle2.with_acquired(move || future::ok::<_, ()>(()));
   runtime.spawn(future::ok::<_, ()>(()).select(waiter).then(move |res| {
-    let mut waiter_fute = match res {
-      Ok((_, fute)) => fute,
+    let mut waiter_future = match res {
+      Ok((_, future)) => future,
```

Comment:
I'm not sure about this family. It's possible that `fute` is a technical term. It's hard to tell if `fute` relates to `futex` or `future` which is why I'm offering a replacement.